PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -21,7 +21,8 @@ public static void Main(string[] args)
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        return (int)((numberCorrectAnswers / (double)numberOfQuestions) * 100) + "%";//Needed to use doubles
```

Comment:
```suggestion
        return (double)((numberCorrectAnswers / numberOfQuestions) * 100) + "%";//Needed to use doubles
```
This can be simplified, and the `(int)` is unnecessary as the number is already an `int`